ocpmodels/models/forcenet.py

```diff
@@ -0,0 +1,476 @@
+import os
+from math import pi as PI
+
+import numpy as np
+import torch
+import torch.nn as nn
+from torch_geometric.nn import MessagePassing, global_add_pool, radius_graph
```

Comment:
We don't seem to be using `global_add_pool` and `radius_graph`

Reply:
removed, thanks.